packages/web3-eth/src/web3_subscriptions.ts

```diff
@@ -58,6 +67,16 @@ export class LogsSubscription extends Web3Subscription<
 	}
 }
 
+/**
+ * ##subscribe(“pendingTransactions”)
```

Comment:
Headers do not work here
<img width="853" alt="image" src="https://user-images.githubusercontent.com/9366141/225754122-382296a9-df3c-42d8-a150-dfc1e972662a.png">

Reply:
fixed